packages/11ty/.eleventy.js

```diff
@@ -147,13 +147,29 @@ module.exports = function(eleventyConfig) {
       /**
        * @see https://vitejs.dev/config/#build-options
        */
+      root: '_site',
       build: {
+        assetsDir: '_assets',
         manifest: true,
         mode: 'production',
+        outDir: '_site',
         rollupOptions: {
-          // plugins: [
-          //   scss() // @see https://github.com/thgh/rollup-plugin-scss
-          // ]
+          output: {
+            assetFileNames: ({ name }) => {
+              const fullFilePathSegments = name.split('/').slice(0, -1)
+              let filePath = '_assets/';
+              ['_assets', 'node_modules'].forEach((assetDir) => {
+                if (name.includes(assetDir)) {
+                  filePath +=
+                    fullFilePathSegments
+                      .slice(fullFilePathSegments.indexOf(assetDir) + 1)
+                      .join('/') +
+                    '/'
+                }
```

Comment:
@anderspollack we might want to rewrite this as a separate method or rollup `plugin` though let's merge this solution.